File: backend/Multibanca.Domain.Models/Multibanca/realizar_control_credito.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class realizar_control_credito : base_auditoria
    {
        public int id_realizar_control_credito { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public bool enviar_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/realizar_control_credito/models/realizar_control_credito.ts
export interface RealizarControlCredito {
  id_realizar_control_credito: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  enviar_reparo: boolean;
  observaciones?: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}